### PR TITLE
Feat/1 add postgis docker deployment

### LINKED ISSUE #1: [Feature] Add PostGIS Docker deployment
Add Docker deployment of PostGIS (PostgreSQL) DB.

### PR DESCRIPTION
**Pull request by nicdub**
First draft
FIX #1

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+include .env
+include secrets/.env
```
Comment (regislon) [inline Makefile]: is there a reason to put 2 env files ?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+run: # main entry for on server, can be run on desktop also
+	docker-compose build --pull
+	docker-compose up --remove-orphans --force-recreate -d
+
+backup-dump: # copy everything you want to backup into ${DUMP_LOCATION}
+	docker-compose exec postgis pg_dumpall -h localhost -U ${POSTGRES_USER} | gzip > ${DUMP_LOCATION}/dump.sql.gz
+
+restore-dump:
+	gunzip ${DUMP_LOCATION}/dump.sql | docker-compose exec -T postgis psql -h localhost -U ${POSTGRES_USER}
```
Comment (regislon) [inline Makefile]: to ease the users process, could we not define by default the SQL dump from the official repo ?

File: database/create_tables.sql [hunk 3]
```diff
@@ -1,8 +1,8 @@
--- Table: tempaqua_sample.anchor_point
+-- Table: public.anchor_point
```
Comment (regislon) [inline database/create_tables.sql:3]: for consitency, I think it's better to keep a distinct schema and not use the public one. What do you think @nicdub

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  postgis:
+    image: postgis/postgis:15-3.3
+    env_file:
+      - ./secrets/.env
+    ports:
+      - "5432:5432"
+    restart: unless-stopped
+    volumes:
+      - database:/var/lib/postgresql/data
+      - ./database/create_tables.sql:/docker-entrypoint-initdb.d/create_tables.sql
```
Comment (regislon) [inline docker-compose.yml]: Nice !

Comment (regislon) [reply docker-compose.yml]: why not puuting the create table also here ? therefore we can just run the docker-compose...

File: secrets/.env_template [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Fill this .env_template with the values required and rename it .env
+
+# POSTGRES_DB=tempaqua_sample
```
Comment (regislon) [inline secrets/.env_template]: DB : tempaqua
user : tempaqua_admin
schema : tempaqua_sample

Open to discuss @nicdub

File: Makefile (hunk 1 above)
Comment (nicdub) [reply Makefile]: this was to separate encrypted .env on ansible, but this is not necessary, you're right, I'll merge to keep it simple.

**Comment by regislon** [comment]
@nicdub I've added the 3 commits as diuscussed today.

**Pull request merged**